src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager_Blocking.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using System.Threading;
+using System;
+using Microsoft.IdentityModel.Logging;
+using Microsoft.IdentityModel.Protocols.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.Telemetry;
+
+namespace Microsoft.IdentityModel.Protocols
+{
+    partial class ConfigurationManager<T> where T : class
```

Comment:
Does it matter if this definition is different than the other?
partial class ConfigurationManager<T> : BaseConfigurationManager, IConfigurationManager<T> where T : class

Reply:
This is the minimum amount needed here.